src/facets/DiamondManagerFacet.sol

```diff
@@ -156,6 +159,19 @@ contract DiamondManagerFacet {
         season.rewardTokenBalance = _rewardTokenToDistribute;
     }
 
+    function claimTokensForSeason() external onlyOwner {
+        IEmissionsManager(s.emissionsManager).mintLiquidStaking();
```

Comment:
```suggestion
        IEmissionsManager(s.emissionsManager).mintLiquidMining();
```

Reply:
I have made this change. I will create separate PR to change function name in EmissionsManager contract.